test/btree2.c

```diff
@@ -9917,19 +9917,21 @@ main(void)
     unsigned         reopen;                    /* Whether to reopen B-tree during tests */
     const char      *driver_name;
     bool             api_ctx_pushed = false; /* Whether API context pushed */
+    int              localTestExpress;       /* localized TestExpress */
 
     driver_name = h5_get_test_driver_name();
 
     /* Reset library */
     h5_test_init();
-    fapl = h5_fileaccess();
+    fapl             = h5_fileaccess();
+    localTestExpress = TestExpress;
 
     /* For the Direct I/O driver, skip intensive tests due to poor performance */
-    if (!strcmp(driver_name, "direct"))
-        SetTestExpress(2);
+    if (localTestExpress < 2 && strcmp(driver_name, "direct"))
+        localTestExpress = 2;
 
-    if (TestExpress > 1)
-        printf("***Express test mode on.  Some tests may be skipped\n");
+    if (localTestExpress > 0)
```

Comment:
Why the change from 1 to 0 here? All the other places in this PR use 1.

Reply:
Technically, level one could skip tests. The other uses actually skip tests if level 2 or 3.

Reply:
> Technically, level one could skip tests. The other uses actually skip tests if level 2 or 3.

Fixed other files.